cfs_config_util/activation.py

```diff
@@ -26,14 +26,15 @@
 """
 import logging
 
-from cfs_config_util.apiclient import APIError, HSMClient
-from cfs_config_util.cfs import (
+from csm_api_client.service.cfs import (
     CFSClient,
     CFSConfigurationLayer,
     CFSConfigurationError,
     LayerState
 )
-from cfs_config_util.session import AdminSession
+from csm_api_client.service.gateway import APIError
+from csm_api_client.service.hsm import HSMClient
+from csm_api_client.session import AdminSession
```

Comment:
Line 82 now has a warning: "Unresolved attribute reference 'get_session' for class 'AdminSession'"

I'm thinking that line must need to do something like you did in `main`:

```
AdminSession(API_GW_HOST, API_CERT_VERIFY)
```